tests/actions/vstudio/cs2005/projectsettings.lua

```diff
@@ -13,17 +13,17 @@
 -- Setup
 --
 
-	local sln, prj
+	local wks, prj
 
 	function suite.setup()
 		_ACTION = "vs2005"
-		sln = test.createsolution()
+		wks = test.createWorkspace()
 		language "C#"
 		uuid "AE61726D-187C-E440-BD07-2556188A6565"
 	end
 
 	local function prepare()
-		prj = premake.solution.getproject(sln, 1)
+		prj = premake.solution.getproject(wks, 1)

```

Comment:
Instead of spamming this, it looks like a lot of the test files for visual studio have this kind of thing. It appears to be intentional. :)